example/debug/odp_debug.c

```diff
@@ -258,12 +266,6 @@ static int queue_debug(void)
 		return -1;
 	}
 
-	/* Configure scheduler before creating any scheduled queues */
-	if (odp_schedule_config(NULL)) {
-		ODPH_ERR("Schedule config failed\n");
-		return -1;
-	}
-
```

Comment:
What's the reason for this removal?

Reply:
It was not removed, but moved into main(). It needs to be called before any test that creates scheduled queues.